mmf/utils/logger.py

```diff
@@ -47,38 +49,49 @@ def __init__(self, config, name=None):
             name = __name__
         self.logger = logging.getLogger(name)
         self._file_only_logger = logging.getLogger(name)
-        warnings_logger = logging.getLogger("py.warnings")
+        self.warnings_logger = logging.getLogger("py.warnings")
 
         # Set level
         level = config.training.logger_level
         self.logger.setLevel(getattr(logging, level.upper()))
         self._file_only_logger.setLevel(getattr(logging, level.upper()))
 
+        # Capture stdout to logger
+        self.stdout_logger = StreamToLogger(
```

Comment:
There should be an option to disable stdout. I don't always want to override my sys.stdout.

Reply:
I will add an override function.